deployments/snapshots/snap_tests.py

```diff
@@ -338,8 +338,8 @@
+0,100000,,,12,12,disaster-type-olabeSHyDByhcbGOMQJZRETyhaJfsmxyYfsdUpJsuPpPfLGWdY,ErAYWXJPimutALuRYgcamUXbDWzlaxrvVLyCevbScBBQyBrfeaPzxtfTfTosaZCIugzNLrXNEcHNaxXJWmBAJmsJjyIUKKNzkyMDYaHoIBJJHOKhEHRQPFYNexbxZLBlxDeKDybSMpibfiBSWwDOHKJlyatfJaWKxieYDxvGuhDFDsTfQViFsyMlYegIRZPJjrfMTLziBrlkjMEvGLeCdZTqUUxrigJwRsxFcuBXGiaKjixFaTKbUdeOzoTvArdDknirElGkJNuUYradfFHPHlkSqTCpfiAeyIYXxaEedYwJPSWdfhKxaNlDZgQfulBfVcVxoYCdKimccDeZBLuRaaEqSPfGKYtJTuSsoZPjZKHDALJKnAwiMqrEXhAUfNYUGjEmZtZIfnXUXecqvcjwXJQNzoRleDsnYYUxCqHWpvjAHdzrIBcVBCmLJoTqtIGmJtluZoyZOtziIdvvdCCrTdzxCNvwfxzknKlUaMcIvXugcFOStUbQ,en,2008-01-01,8,10,,8,event-HSghViBXbcmdHnxyzMfWbyjRDeuVnglXnqvwdBGJlBKgFOFBWJ,7,pjnnhqsvchptjayhcxmvtpeqjvhiacnopxnuhdgqxcssbmdgxk,,en,4,2008-01-01T00:00:00.123456Z,,,project-MgogfgDXGygMRbjvZsTCRydRSlkObzTNxourSfRlafQilCuYYK,0,Programme,1,sect-uVAlmiYIxHGrkqEZsVvZhDejWoRURzZJxfYzaqIhDxRVRqLyOx,1,Multilateral,9,9,lF,DQz,country-DjTjcfdLhjgCijUesXvSuLpTEBAhPQQJULtTnEPcxkzmwedcZq,society-name-YTuCiuVTBQuWKSlaYkqAfFvjcipGMFrmUlAcXaOxTvZNxvagEX,"CefuMjeirN, QWBUdsMtwg","1, 2","False, True","False, True","district-hQbiawYYpLublqdiVAHhVeECXxGLgCGoNcUYQHtDPbdEzBRgFT, district-gYShPPXJUnBCoAvDzAUguBuQqxjREefffBgVVxZiJdLJJvQhAw",0,0,0,0,4,2008-01-01T00:00:00.123456Z,4,2008-01-01T00:00:00.123456Z,regional-project-ZPRRLqCONvcCscFgWCTmiDMJdwTVLRxPboQWedazsRNntXUanS,en,False,8,,,,8,FE,JMs,country-QGXznlhcyZgVnmWKVVtSRnLtxDdPOoKqnGbtanlvnFUyUuDzMW,society-name-YQpeNgabNdaratCyUKkgdxnPmBVxerAPbtXchtjqDZnhOVRYbE,"1, 2","sect-tag-RoEbNJuNoPeODStPAhicctFhgpIiyDxQVSIALVUjAPgFNArcSx, sect-tag-CCpxgRxIPwdzrmhDfQnPOMbdYvpiYKneWJnLnovXjYMarjiIqZ",2008-01-01,1,Ongoing,0,0,0,0,en,False,8,public,Public\r
```

Comment:
Let's just change it from here.

https://github.com/IFRCGo/go-api/blob/721f555dffd2a74ed3ffd8859d491a2dd85d4c5a/deployments/tests.py#L190-L208


For reference, 
Changes done in this PR: https://github.com/IFRCGo/go-api/pull/2309/files#diff-7d1870db44999b07b65df11ee4bd4f60f5a8644d97eea7f03f6ae652de60b036R194

Reply:
Not crystalclear what to do. 
(Anyway, snapshot test changes are not a huge tasks, just simply remove the main content of the shapshot/snap_tests.py and it is regenerated.)
Anyway, if you've already done a change in another branch, please do it in this one also, @susilnem .

Reply:
Thanks so much.